Dizzy/UI Components/LocationLabel.swift

```diff
@@ -11,10 +11,13 @@ import SnapKit
 
 class LocationLabel: UIView {
     
-    let textLabel = UILabel()
-    let horizontalPadding = CGFloat(4)
-    let cornersRadius = CGFloat(13)
-    let backgroundAlpha = CGFloat(0.5)
+    private let textLabel = UILabel()
+    private let bedgeButton = UIButton().smallRoundedBlackButton
+    private let horizontalPadding = CGFloat(4)
```

Comment:
you should write whos horisontal padding is it (textLabelHorizontalPadding)

Reply:
Done